packages/zowe-explorer/CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 - Added option to save unique data set attributes as a template after allocation for future use. [#1425](https://github.com/zowe/vscode-extension-for-zowe/issues/1425)
 - Added "Cancel Job" feature for job nodes in Jobs tree view. [#2251](https://github.com/zowe/vscode-extension-for-zowe/issues/2251)
+- Added "Sort Jobs" feature for job nodes in Jobs tree view. [#2257](https://github.com/zowe/vscode-extension-for-zowe/issues/2251)
```

Comment:
should move CHANGELOG item up under `TBD Release` section

Reply:
Modified the CHANGELOG item